sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Reshuffle.java

```diff
@@ -109,16 +110,33 @@ public void processElement(
   public static class ViaRandomKey<T> extends PTransform<PCollection<T>, PCollection<T>> {
     private ViaRandomKey() {}
 
+    private ViaRandomKey(@Nullable Integer numBuckets) {
+      this.numBuckets = numBuckets;
+    }
+
```

Comment:
Could you send the update to core as a separate PR ?

Reply:
These are only minor changes to the transform and this is the only file not directly related to pubsub lite io that is changed. Given the high latency on merging beam PRs and the fact that this change was proposed in this pr (see millsd@ comments above), I'd prefer not to.